wallet/package.json

```diff
@@ -64,8 +68,11 @@
         "webpack-merge": "^5.8.0"
     },
     "dependencies": {
+        "@mysten/sui.js": "file:../sdk/typescript",
+        "bip39-light": "^1.0.7",
```

Comment:
We could have used bip39, but I ended up using bip39-light because the former contains the wordlist from a lot of other languages as well and have more extra dependencies that we don't need. The latter is also used by Near